packages/proto-signing/src/testdata/coins.json

```diff
@@ -0,0 +1,50 @@
+[
+  [
+    [{ "amount": "1", "denom": "ucosm" }, { "amount": "3", "denom": "ustake" }],
+    {
+      "ucosm": { "denom": "COSM", "exponent": 6 },
+      "ustake": { "denom": "STAKE", "exponent": 6 }
+    },
+    "0.000001 COSM, 0.000003 STAKE"
+  ],
+  [
+    [{ "amount": "3", "denom": "ustake" }, { "amount": "1", "denom": "ucosm" }],
+    {
+      "ucosm": { "denom": "COSM", "exponent": 6 },
+      "ustake": { "denom": "STAKE", "exponent": 6 }
+    },
+    "0.000001 COSM, 0.000003 STAKE"
+  ],
+  [
+    [{ "amount": "1", "denom": "uaa" }, { "amount": "2", "denom": "ubb" }, { "amount": "3", "denom": "uatom" }],
+    {
+      "uaa": { "denom": "AA", "exponent": 6 },
+      "ubb": { "denom": "BB", "exponent": 6 },
+      "uatom": { "denom": "atom", "exponent": 6 }
+    },
+    "0.000001 AA, 0.000002 BB, 0.000003 atom"
+  ],
+  [
+    [
+      { "amount": "8", "denom": "_uxc1" },
+      { "amount": "7", "denom": "_uxc" },
+      { "amount": "6", "denom": "_uxb" },
+      { "amount": "5", "denom": "_uxa" },
```

Comment:
I got an error here on the SDK: these are Coins, so their denom can't start with `_`. Maybe we can add the `_` inside the denom?

Reply:
Sure, done